index.bs

```diff
@@ -16,69 +16,75 @@ Indent: 2
-  to focus the editor, working group attention, and reviewers' attention. As such,
-  the questionnaire is evidence and research based and provides examples and past cases.
-
-The audience of this document is general:
-
-  *   The editors and contributors who are responsible for the development of the feature,
-  *   The W3C TAG, who receive the questionnaire along with the request, and in line with the W3C Process,
-  *   External audience (developers, designers, etc.) wanting to understand the possible security and privacy implications.
+  When designing new features for the Web platform,
+  it's vital that we consider the security and privacy implications of our work,
+  and strive to make design decisions that
+  enhance rather than degrade
+  the security and privacy characteristics of the Web.
+
+  This document contains a set of questions
+  intended to help <abbr title="specification">spec</abbr> authors
+  as they think through
+  the security and privacy implications
+  of their work.
+  It also documents mitigation strategies
+  that spec authors can use to address
+  security and privacy concerns they encounter as they work on their spec.
+
+  This document is itself a work in progress,
+  and there may be security or privacy concerns
+  which this document does not (yet) cover.
+  Please [let us know](https://github.com/w3ctag/security-questionnaire/issues/new)
+  if you identify a security or privacy concern
+  this questionnaire should ask about.
+
+  <h3 id="howtouse">How To Use The Questionnaire</h3>
+
+  Spec authors should work through these questions
+  early on in the design process,
+  when things are easier to change.
+  When privacy and security issues are only found later,
+  after the feature has shipped
+  and when it's much harder to change the design,
+  user agents may need to adopt breaking changes
+  to protect their users' privacy and security.
+
+  These questions should be kept in mind throught work on a specification.
+  Spec authors should periodically revisit this questionnaire
+  to freshly consider the privacy and security implications of their specification
+  as their feature's design changes over time.
+
+  <h3 id=reviews>TAG and PING reviews and this questionnaire</h3>
+
+  When authors request
+  a [specification review](https://github.com/w3ctag/design-reviews)
+  from the [Technical Architecture Group (TAG)](https://www.w3.org/2001/tag/),
+  the TAG asks that authors provide answers
+  to the questions in this document.
+  The [Privacy Interest Group (PING)](https://www.w3.org/Privacy/IG/)
+  also considers answers to these questions
+  while conducting
+  [privacy reviews](https://github.com/w3cping/privacy-reviews/issues).
+
+  The TAG and PING use this document
+  to record security and privacy questions
+  which come up during our spec reviews.
+  Working through these questions can save
+  both spec authors and the folks performing design reviews
```

Comment:
```suggestion
  both spec authors and the people performing design reviews
```

I'm generally pro "folks" but it might be unfamiliar, and it's a bit of a change in register here.